packages/cfgen/cfgen.1.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Daniil Baturin <daniil@baturin.org>"
+authors: "Daniil Baturin <daniil@baturin.org>"
+homepage: "https://github.com/dmbaturin/cfgen"
+bug-reports: "https://github.com/dmbaturin/cfgen/issues"
+license: "MIT"
+dev-repo: "https://github.com/dmbaturin/cfgen"
```

Comment:
Yes, I noticed this already, and fixed in the repo (I went for git+https://), and then re-run opam-publish submit, but it looks like I did something wrong, I assumed that it will automatically update the opam file in the pull request.
Fixed it in the pull request by hand.